package.json

```diff
@@ -48,46 +46,10 @@
   },
   "homepage": "https://github.com/Seneca-CDOT/telescope#readme",
   "dependencies": {
-    "@bull-board/api": "3.10.2",
-    "@bull-board/express": "3.10.2",
-    "@elastic/elasticsearch": "7.16.0",
-    "@elastic/elasticsearch-mock": "0.3.1",
-    "@wordpress/wordcount": "2.15.2",
     "babel-jest": "27.5.1",
-    "bull": "3.29.3",
-    "clean-whitespace": "0.1.2",
-    "connect-redis": "6.1.3",
-    "cors": "2.8.5",
-    "date-fns": "2.28.0",
     "docker-compose": "0.23.17",
     "dotenv": "10.0.0",
-    "entities": "3.0.1",
-    "express": "4.17.3",
-    "express-healthcheck": "0.1.0",
-    "express-session": "1.17.2",
-    "express-validator": "6.14.0",
-    "feed": "4.2.2",
-    "helmet": "4.6.0",
-    "highlight.js": "11.4.0",
-    "http-proxy-middleware": "2.0.4",
-    "ioredis": "4.28.5",
-    "ioredis-mock": "5.9.1",
-    "jsdom": "18.1.1",
-    "linkify-html": "3.0.5",
-    "linkifyjs": "3.0.5",
-    "node-fetch": "2.6.7",
-    "normalize-url": "6.1.0",
-    "opml-generator": "1.1.1",
-    "passport": "0.5.2",
-    "passport-saml": "3.2.1",
-    "pino": "7.8.0",
-    "pino-elasticsearch": "6.2.0",
-    "pino-http": "6.6.0",
-    "pino-pretty": "7.5.3",
-    "rss-parser": "3.12.0",
-    "sanitize-html": "2.6.1",
-    "set-interval-async": "2.0.3",
-    "stoppable": "1.1.0"
+    "supertest": "6.1.6"
```

Comment:
`supertest` is already in `devDependencies` (line 118), was the one under `dependencies` added by accident?